steps/run-android-instrumentation-tests/1.0.0/step.yml

```diff
@@ -0,0 +1,136 @@
+  commit: b4e5502f25bbec6c9b1db6f7f519b0361b293ad3
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  apt_get:
+  - name: curl
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: emulator.wtf API Token
+  token: null
+- app: $BITRISE_APK_PATH
+  opts:
+    description: |
+      The path to the APK to test.
+
+      When you run the [android-build](https://www.bitrise.io/integrations/steps/android-build) or [android-build-for-ui-testing](https://bitrise.io/integrations/steps/android-build-for-ui-testing) Step before this Step, then the generated APK path is exported under the `$BITRISE_APK_PATH` environment variable.
+    is_expand: true
+    is_required: true
+    summary: The path to the APK to test.
+    title: APK path
+- opts:
+    description: |
+      The path to the test APK that contains instrumentation tests.
+
+      You can easily generate the test APK by running the [android-build-for-ui-testing](https://bitrise.io/integrations/steps/android-build-for-ui-testing) Step before this Step. In this case, the test APK path is exported under the `$BITRISE_TEST_APK_PATH` environment variable.
+    is_expand: true
+    is_required: true
+    summary: The path to the test APK that contains instrumentation tests.
+    title: Test APK path
+  test: $BITRISE_TEST_APK_PATH
+- num_shards: 1
+  opts:
+    description: |
+      Tests are split evenly by count across shards and run in parallel.
+
+      In general, the more shards you use, the faster your total test run time will be. Make sure not to set this count to be bigger than your actual test count.
+    summary: Tests are split evenly by count across shards and run in parallel.
+    title: Number of test shards
+- device: null
+  opts:
+    description: |
+      Device configuration where to run tests.
+
+      This is only required when you don't want to run tests on the default emulator configuration.
+
+      The value of this parameter must be a list of key=value pairs where supported keys are:
+
+      -  `model`: device model to use, possible values [`Pixel2`, `NexusLowRes`]
+      -  `version`: API version to use
+
+      For example, to test on an low res MDPI device with API 23 specify `model=NexusLowRes,version=23`
+    summary: Device configuration where to run tests
+    title: Device configuration
+- opts:
+    title: Use Android Test Orchestrator
+    value_options:
+    - "yes"
+    - "no"
+  use_orchestrator: "yes"
+- clear_package_data: "yes"
+  opts:
+    summary: Only applicable when using Android Test Orchestrator
+    title: Clear package data between runs
+    value_options:
+    - "yes"
+    - "no"
+- opts:
+    title: Collect code coverage data while executing the test
```

Comment:
Please add `summary` to this input. (All inputs should have title and summary at least, while description is optional).